src/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj

```diff
@@ -328,6 +358,11 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="project.json" />
+
+    <!-- TODO: Replace this reference with package reference -->
+    <ProjectReference Include="..\..\System.ComponentModel.Primitives\ref\System.ComponentModel.Primitives.csproj"/>
+    <!-- TODO: Replace this reference with package reference -->
+    <ProjectReference Include="..\..\System.ComponentModel.TypeConverter\ref\System.ComponentModel.TypeConverter.csproj"/>
```

Comment:
Initially this was because we needed it in the implementation of the SynchronizingObject property. However, we are not going to use the implementation of that from Desktop so this dependency is not longer necessary.